force-app/main/default/classes/MA_ReportSourceBatchableTest.cls

```diff
@@ -246,10 +248,11 @@ private class MA_ReportSourceBatchableTest {
             // because the batchable throws the error in the start method
             // then we're getting a synchronous exception caught rather than
             // the job's finish method logging the error asynchronously
-            System.assert( ex.getMessage().containsIgnoreCase( 'The data you’re trying to access is unavailable' ), 'failed for wrong reason' );
-
+            noDataFoundExceptionThrown = true;
```

Comment:
Agree, I like the simplification change here of just relying on the caught exception type and not inspecting the message.